app/models/pipeline_run.rb

```diff
@@ -126,25 +120,25 @@ class PipelineRun < ApplicationRecord
   STATUS_RUNNABLE = 'RUNNABLE'.freeze # TODO: (gdingle): not used anywhere?
   STATUS_READY = 'READY'.freeze
   # NOTE: The current stored job_status are...
-  # +-------------------------------------------+
-  # | job_status                                |
-  # +-------------------------------------------+
-  # | 1.Host Filtering-FAILED                   |
-  # | 1.Host Filtering-FAILED|READY             |
-  # | 2.GSNAPL/RAPSEARCH alignment-FAILED       |
```

Comment:
this list is supposed to represent historical values, not current ones––if those are even changing